packages/x-charts/src/models/axis.ts

```diff
@@ -333,6 +333,13 @@ export type AxisConfig<
    * If `true`, Reverse the axis scaleBand.
    */
   reverse?: boolean;
+  /**
+   * Defines the axis scale domain based on the min/max values of series linked to it.
+   * - 'nice': Rounds the domain at human friendly values.
+   * - 'strict': Set the domain to the min/max value provided. No extras space is added.
+   * - function: Takes the extremums to display as an arrgument, and return the axis domain.
+   */
+  domainLimit?: 'nice' | 'strict' | ((range: [number, number]) => [number, number]);
```

Comment:
Should the func signature be clearer? eg: `(min: number, max: number) => {min: number, max: number}`

Reply:
I don't have a strong opinion on this one. The reason why I proposed a signature `([number, number]) =>[number, number]` is to match the D3 representation of range and domain.

Reply:
Yes, but then you have to be aware of the d3 representation, while `min/max` removes that overhead. 

I don't think it is a big deal too, but it could be simpler for users.

Reply:
Expecting a different format makes the code a bit more complex (need to reshape the object to an array) but that's ok